flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/plan/QueryOperationConverter.java

```diff
@@ -334,6 +343,68 @@ public RelNode visit(FunctionQueryOperation functionTable) {
                                                             new int[0]);
                                             inputStack.add(relBuilder.build());
                                             return tableArgCall;
+                                        } else if (resolvedArg
```

Comment:
I don't think it's a good idea to use instance of here. It's better we reuse ExpressionConverter to convert these expression. How about let `ExpressionConverter`extends `ResolvedExpressionVisitor<RexNode>`?